classes/classes/Scenes/Inventory.as

```diff
@@ -697,8 +697,8 @@ use namespace CoC;
 				freeRows = 6 - (playerItemCount + 4) / 5;
 				N += freeRows*5;
 			}
-			var playerPageMax:int = Math.max(0, (playerItemCount - 1) / N);
-			var storagePageMax:int = Math.max(0, (storageItemCount - 1) / N);
+			var playerPageMax:int = Math.max(0, Math.ceil((playerItemCount - 1) / N));
+			var storagePageMax:int = Math.max(0, Math.ceil((storageItemCount - 1) / N));

```

Comment:
Good catch, but you've made a small mistake.
Say N is 10 (for simplicity), storageItemCount is 21.
Then `storagePageMax` would be 2 (should be 3).

Actually, the original code for last page number was corect, the error was in application: `.disableIf(playerPage == storagePageMax-1)` - it was written as if `storagePageMax` kept __number of pages__, not __number of last page__.
So to fix it, you need either:
1) Remove `-1` from `storagePageMax` - so it would calculate correct number of pages.
2) Keep old formula, and remove `-1` from button disable code.